process_report/invoices/NERC_total_invoice.py

```diff
@@ -64,5 +60,5 @@ def _lower_col(data):
         ]
         self.export_data = self.export_data[
             self.export_data[invoice.INSTITUTION_FIELD].isin(included_institutions)
-            | (self.export_data[invoice.GROUP_MANAGED_FIELD].apply(_lower_col) == "yes")
+            | (self.export_data[invoice.GROUP_MANAGED_FIELD] == True)  # noqa: E712
```

Comment:
why did you disable E712 instead of doing what it actually says i.e. `something is True` instead of `something == True`?

Reply:
I somehow forgot that was an option :P